app/assets/javascripts/components/features/compose/components/compose_form.jsx

```diff
@@ -18,7 +18,8 @@ import TextIconButton from './text_icon_button';
 
 const messages = defineMessages({
   placeholder: { id: 'compose_form.placeholder', defaultMessage: 'What is on your mind?' },
-  spoiler_placeholder: { id: 'compose_form.spoiler_placeholder', defaultMessage: 'Content warning' },
+  hide_text_placeholder: { id: 'compose_form.hide_text_placeholder', defaultMessage: 'This text hide by default.' },
+  spoiler_placeholder: { id: 'compose_form.spoiler_placeholder', defaultMessage: 'Write warnings before hide text.' },
```

Comment:
Minor change, for English should be "Warning before hidden text" or "Write warning before hidden text"